src/main/java/frc/robot/Constants.java

```diff
@@ -163,4 +164,24 @@ public static class enabledSubsystems {
 
   // Lights
   public static final int LED_COUNT = 20;
+
+  public static class ClimberConstants {
+    public static final double GEAR_RATIO = 0;
+    public static final double MAX_ERROR = 0;
+    public static final double PICK_UP = 0;
+    public static final double CLIP_VOLTAGE = 0;
```

Comment:
What does CLIP_VOLTAGE refer to?

Reply:
the constant voltage used while the climber arm is lowering onto the chain. We are using a constant voltage instead of a set position for this specific step in the climb process because the height of the chain varies depending on where we try to climb from, so setting a specific position will not work well.